src/main/java/edu/kis/powp/jobs2d/events/SelectOpenFromJsonOptionListener.java

```diff
@@ -22,30 +17,14 @@ public void actionPerformed(ActionEvent e) {
         chooser.setCurrentDirectory(workingDirectory);
         if (chooser.showOpenDialog(null) == JFileChooser.APPROVE_OPTION) {
             File f = chooser.getSelectedFile();
-            System.out.println(f.getName());
-            drawFromJson(f.getPath());
+            String command;
+            try {
+                command = new String(Files.readAllBytes(Paths.get(f.getPath())));
+                Loader loader = new LoadFromJson();
```

Comment:
I think that the listener could be also reusable for other formats:
- loader should be injected
- the name SelectOpenFromJsonOptionListener should be more like: SelectImportCommandFromFileOptionListener